README.md

```diff
@@ -54,9 +54,9 @@ Schemable combinator modules must have a unique name and can be found in the fol
 
 ### Generating SchemableExt/Instances
 
-Once the new combinator modules are in place, run `yarn generate` and the ts script will update the typeclass instances and SchemableExt module with the newly created combinators.
+Once the new combinator modules are in place, run `yarn generate:schemables` and the ts script will update the typeclass instances and SchemableExt module with the newly created combinators.
 
-_Coming soon: convenience function for generating new Schemable combinators._
+Create a new combinator file using `yarn generate:template [string | number] [NameOfCombinator]`. Note: the name of the new combinator must be unique, and begin with a capital letter.
```

Comment:
Silly nitpick, but it might be a little more logical to switch these two paragraphs.